app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -6,6 +6,7 @@
   label ||= false
   name ||= id
   is_page_heading ||= false
+  data_attributes ||= { }
```

Comment:
Just a minor comment - while this doesn't seem to impact passing data attributes to the component, I think these are written as `data_attributes ||= {}` elsewhere in the repo e.g. [error alert component](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_error_alert.html.erb#L6)